.github/workflows/deploy-gh-pages.yml

```
@@ -0,0 +1,27 @@
+on: [push]
+
+jobs:
+  check-clean-style:
```

(Nitpick) this name is copied over from another workflow and we should change it to something more descriptive eg:

```suggestion
  deploy:
```

---

Indeed you are right! Resolved!